code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -223,6 +223,11 @@
 			if(o.applied_pressure == src)
 				msg += "<span class='warning'>[T.He] [T.is] applying pressure to [T.his] [o.name]!</span>\n"
 
+	//moisture
+	if(istype(ST))
+		if(ST.moisture < MOISTURE_LOW)
+			msg += "<span class='warning'>[T.He] looks dried out!</span>\n"
```

Comment:
hate to be this girl but please follow the SPAN_X standards

Reply:
Following the standards set by the actual proc, futhuremore, using SPAN_X()+"\n" is unneeded overhead.